rosetta/src/main/java/com/ahmedjazzar/rosetta/LanguagesListDialogFragment.java

```diff
@@ -153,7 +153,8 @@ protected void onPositiveClick() {
                     getActivity(), mSelectedLanguage)) {
 
                 mLogger.info("App locale changed successfully.");
-                LocalesUtils.refreshApplication(getActivity());
+                //LocalesUtils.refreshApplication(getActivity());
```

Comment:
Just remove it

Reply:
when i was using your refresh method the problem was that it was recreating application and current activity both.
i just wanted to recreate whole application. so it can recreate all resources in my application

Reply:
Cool with me, but just no need to comment, removing the line is enough.